cli/command/cli.go

```diff
@@ -396,7 +406,7 @@ func (cli *DockerCli) StackOrchestrator(flagValue string) (Orchestrator, error)
 	if currentContext == "" {
 		currentContext = configFile.CurrentContext
 	}
-	if currentContext != "" {
+	if currentContext != "" && currentContext != DefaultContextName {
```

Comment:
doesn't `ContextStoreWithDefault` now also implement `GetContextMetadata()`? IOW; do we need to still have this special case?